fineract-provider/src/main/java/org/apache/fineract/portfolio/self/security/api/SelfUserApiResource.java

```diff
@@ -59,7 +62,12 @@ public SelfUserApiResource(final UsersApiResource usersApiResource,
         }
 
         @PUT
-        public String update(final String apiRequestBodyAsJson) {
+        @Consumes({ MediaType.APPLICATION_JSON })
+        @Produces({ MediaType.APPLICATION_JSON })
```

Comment:
why have you added this line?
not in original code

Reply:
Giving error!
not generating SwaggerJson and UI

Reply:
@Sanyam96 I am not getting a single error while building this or running this.

Reply:
@Sanyam96 I check each and every file before opening a PR

Reply:
![capture](https://user-images.githubusercontent.com/20682192/34324254-bf3a912a-e891-11e7-9a88-aac1db3d5cd6.PNG)


![capture](https://user-images.githubusercontent.com/20682192/34324257-d60cd9c6-e891-11e7-9943-9d71d2406b28.PNG)

Reply:
Here you can see, I can run it successfully.